wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+NavigationBar/FilterButtonStyleHelper.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+struct FilterButtonStyleHelper {
+
+    /// Configures the action image based on the image name and selection state.
+    ///
+    /// - Parameters:
+    ///   - imageName: The name of the image.
+    ///   - isSelected: A boolean indicating whether the filter is currently selected.
+    /// - Returns: A configured `UIImage`.
+    static func configureActionImage(named imageName: String, isSelected: Bool) -> UIImage? {
+        let font = UIFont.systemFont(ofSize: 17)
```

Comment:
Should this font be harded coded with this specification or should we use our default ones from `FontScheme`?

Reply:
For now it will be hardcoded but I plan to introduce a new way of setting the proper font size for SF symbols. FontScheme should be avoided since we're trying to move away from it.